iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -437,6 +441,9 @@ private void commitTable(FileIO io, ExecutorService executor, OutputTable output
       dataFiles.addAll(writeResults.dataFiles());
       deleteFiles.addAll(writeResults.deleteFiles());
       referencedDataFiles.addAll(writeResults.referencedDataFiles());
+      rewritePolicy = rewritePolicy == RewritePolicy.DEFAULT ?
```

Comment:
Because this assignment is happening inside the loop which iterates over the list of job contexts.
I am assigning the rewrite policy in case it is still default (unset) because if there are >1 job context, which one of them will contain the rewrite policy config.

Reply:
in case of IOW, should would be just 1 job context

Reply:
Done.